src/components/NewItemRow/index.jsx

```diff
@@ -0,0 +1,116 @@
+import {useEffect, useState} from "react";
+
+function NewItemRow({
+                        index,
+                        invoiceObj,
+                        setInvoiceObj,
+                    }) {
+
+    const [itemInfo, setItemInfo] = useState({
+        description: "",
+        quantity: 0,
+        rate: 0.00,
+        total: 0.00,
+    });
+    const rowTotal = (quantity, rate) => {
+        return quantity && rate
+            ? (Number(quantity) * Number(rate)).toFixed(2)
+            : (0).toFixed(2);
+    };
+
+    return (
+        <div className="d-flex flex-row w-100 small gap-1">
+            <div className="col-3">
+        <textarea
+            name="textarea"
+            className="w-100"
+            rows="4"
+            placeholder="Optional description"
+            value={itemInfo["description"]}
```

Comment:
While you can use bracket notation to get a JS object value, you've used dot notation elsewhere - pick one and stay consistent.